include/mbedtls/ssl.h

```diff
@@ -1291,7 +1291,23 @@ struct mbedtls_ssl_config
 
 struct mbedtls_ssl_context
 {
-    const mbedtls_ssl_config *MBEDTLS_PRIVATE(conf); /*!< configuration information          */
+    /*
+     * Secure renegotiation
```

Comment:
This comment does not match what follows, it should be "DTLS connection ID".

Reply:
Oh, looking at where this comes from I understand the confusion: there should have been a comment in the first place to make it clear that it was its own thing, not a continuation of secure renego.

Reply:
Done.